src/main/java/model/Cars.java

```diff
@@ -0,0 +1,46 @@
+package model;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class Cars {
+
+    private final List<Car> cars = new ArrayList<>();
+    private List<Car> winners = new ArrayList<>();
+
+    public Cars(final String[] carNames) {
+        for (String carName : carNames) {
+            cars.add(new Car(carName));
+        }
+    }
+
+    public void move() {
+        for (Car car : cars) {
+            car.move();
+        }
+    }
+
+    public void determineWinner() {
+        this.winners = cars.stream().filter(car -> car.getScore() == getMaxScore())
+                .collect(Collectors.toList());
```

Comment:
아래처럼 가독성을 높힐 수 있을 것 같아요.
메서드 체이닝의 경우 줄바꿈을 통해 절차를 표현하면 잘 보이더라고요!
- [디미터의 법칙](https://mangkyu.tistory.com/147) 요런 글도 참고해보시면 좋아요!
```suggestion
        this.winners = cars.stream()
                .filter(car -> car.getScore() == getMaxScore())
                .toList();
```

Reply:
좋은 학습 내용 주셔서 감사합니다!